test/gtest/uct/uct_test.cc

```diff
@@ -227,9 +249,23 @@ void uct_test::set_interface_rscs(const md_resource& md_rsc,
     }
 }
 
-void uct_test::set_sockaddr_resources(const md_resource& md_rsc, uct_md_h md,
-                                      ucs_cpu_set_t local_cpus,
-                                      std::vector<resource>& all_resources) {
+bool uct_test::is_interface_usable(struct ifaddrs *ifa, const char *name) {
```

Comment:
maybe `<check|validate>_ifaddr`?

Reply:
can you please explain why is_interface_usable isn't good?
i think it makes more sense to use "is_" in the function name and then use it in an "if" statement.

Reply:
it's minor, just not clear how iface can be unusable :)

Reply:
if it's not active or doesn't have a valid sa_family (for example) then we can't use it for the test.